test/map-listener-tests.js

```diff
@@ -428,6 +420,15 @@ describe('Map Events IT Test Suite', function () {
       this.updateEvents = []
       this.deleteEvents = []
       this.eventOrder = []
+      this.on(event.MapEventType.DELETE, (event) => {
+        this.entryDeleted(event)
+      })
+      this.on(event.MapEventType.INSERT, (event) => {
+        this.entryInserted(event)
+      })
+      this.on(event.MapEventType.UPDATE, (event) => {
+        this.entryUpdated(event)
+      })
```

Comment:
What's the value of delegation to different methods here? Can't you just implement necessary logic directly within each handler?

At the very least, this could be simplified into:
```javascript
this.on(event.MapEventType.DELETE, this.entryDeleted)
this.on(event.MapEventType.INSERT, this.entryInserted)
this.on(event.MapEventType.UPDATE, this.entryUpdated)
```